architect_py/tests/conftest.py

```diff
@@ -34,3 +36,18 @@ async def async_client():
         host=host, port=port, api_key=api_key, api_secret=api_secret
     ) as client:
         yield client
+
+
+def pytest_addoption(parser):
```

Comment:
do we need this? since its a marker already, i would've thought `pytest -m live_orderflow` would already cover this

Reply:
one would assume that would work but the default behavior seems to be that if you don't specify any markers it will run all tests. Bit strange but I followed the docs here to make it skip if you don't purposefully specify the arg in the cmd line:
https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option